server/sonar-server/src/main/java/org/sonar/server/computation/step/ComputationSteps.java

```diff
@@ -41,6 +41,7 @@
       ParseReportStep.class,
 
       // Persist data
+      ComputeAndPersistDaysSinceLastCommitStep.class,
```

Comment:
Compute is the "Decorator" part and persistence is the persistence part … for the time being we could consider 2 separate operations. WDYT ?